DataFormats/HcalDetId/interface/HcalDetId.h

```diff
@@ -13,22 +13,22 @@
 class HcalDetId : public DetId {
 
 public:
-  static const int kHcalPhiMask1       = 0x7F;
-  static const int kHcalPhiMask2       = 0x3FF;
-  static const int kHcalEtaOffset1     = 7;
-  static const int kHcalEtaOffset2     = 10;
-  static const int kHcalEtaMask1       = 0x3F;
-  static const int kHcalEtaMask2       = 0x1FF;
-  static const int kHcalZsideMask1     = 0x2000;
-  static const int kHcalZsideMask2     = 0x80000;
-  static const int kHcalDepthOffset1   = 14;
-  static const int kHcalDepthOffset2   = 20;
-  static const int kHcalDepthMask1     = 0x1F;
-  static const int kHcalDepthMask2     = 0xF;
-  static const int kHcalDepthSet1      = 0x1C000;
-  static const int kHcalDepthSet2      = 0xF00000;
-  static const int kHcalIdFormat2      = 0x1000000;
-  static const int kHcalIdMask         = 0xFE000000;
+  static const uint32_t kHcalPhiMask1       = 0x7F;
```

Comment:
@bsunanda @Dr15Jones I assume these declarations could become constexpr uint32_t, although not necessary